protos/feast/core/IngestionJob.proto

```diff
@@ -0,0 +1,65 @@
+//
+// Copyright 2020 The Feast Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+syntax = "proto3";
+
+package feast.core;
+
+option go_package = "github.com/gojek/feast/sdk/go/protos/feast/core";
+option java_outer_classname = "IngestionJobProto";
+option java_package = "feast.core";
+
+import "feast/core/FeatureSet.proto";
+import "feast/core/Store.proto";
+import "feast/core/Source.proto";
+
+// Represents Feast Injestion Job 
+message IngestionJob {
+    // Job ID assigned by Feast
+    string id = 1;
+    // External job ID specific to the runner.
+    // For DirectRunner jobs, this is identical to id. For DataflowRunner jobs, this refers to the Dataflow job ID.
+    string external_id = 2;
+    IngestionJobStatus status = 3;
+    // List of feature sets whose features are populated by this job.
+    repeated feast.core.FeatureSet feature_sets = 4;
+    // Source this job is reading from.
+    feast.core.Source source = 5;
+    // Store this job is writing to.
+    feast.core.Store store = 6;
+}
+
+// Status of a Feast Ingestion Job
+enum IngestionJobStatus {
```

Comment:
Could you possibly make feast.core.model.JobStatus a facade of this? or update the Job model to use this enum instead of our own

Reply:
I added `toIngestionProto()` to `JobStatus` that creates the `IngestionJobStatus` proto.